posts/2022/esds-fall-event.md

```diff
@@ -0,0 +1,35 @@
+---
+author: Julia Kent
+date: 2022-10-10
+tags: tutorial hackathon events
+---
+
+# ESDS Fall Event
+
+ESDS is hosting a fall event **November 9th and 10th** (Wednesday-Thursday). This event will be hybrid, in-person in NCAR's Mesa Lab Main Seminar Room and virtual over Zoom.
+
+The ESDS Fall Event will cover a (re)introduction to ESDS and Python tutorials (open to the community). Particular emphasis will be placed on the skills needed to best leverage the GeoCAT Python stack. The event will culminate in a hackathon (NCAR/UCAR personnel only) at the end of the second day.
+
+Here is an approximate content schedule:
+
+Day 1:
+
+- Intro to ESDS
+- Python Intro and Workspace Set Up
+- Python Syntax Basics
+- Numpy
+- Data Visualization
+- Xarray
+- Dask
```

Comment:
Should we put any forum, Q&A to the end of day1? cc @matt-long @katiedagon @clyne

Reply:
@dcherian @hCraker @dcamron and myself are discussing making these events a "flipped classroom" so we request that participants watch an existing tutorial recording and then the session is all Q&A.

Reply:
Oh, yeah, flipped classroom idea for this morning part sounds great! I was referring to more of an ESDS-wise forum or Q&A in the afternoon to listen to the audience about ESDS efforts and events so far, ESDS future, maybe discuss the answers to our questions that we will put into the registration link, etc.